lib/index.js

```diff
@@ -114,27 +115,21 @@ Nanigans.prototype.track = function(track) {
       params.products.value = data.value;
       params.products.sku = data.sku;
       params.products = qs.stringify(params.products);
-      this.load('purchase', params);
+      params.userId ? this.load('purchase', params) : this.load('purchase_no_user_id', params);
       break;
     case 'user':
       switch (event.name) {
-      case 'product':
```

Comment:
@hankim813 - I ran this mongo query `db.integrations.find({type: 'Nanigans'})` and looked at all the records, and no one is using a Nanigans event name ([seen here](https://cl.ly/1i163y0w1N1T)) of `product`. So I think we're okay removing it.